examples/run/run.cpp

```diff
@@ -976,8 +976,11 @@ static int generate(LlamaData & llama_data, const std::string & prompt, std::str
     return 0;
 }
 
+static const char * prompt_prefix_env = std::getenv("LLAMA_PROMPT_PREFIX");
```

Comment:
Can we not use global variables and put these in the one function where they are needed?

Reply:
sure

Reply:
done